sdk/search/Azure.Search.Documents/src/Spatial/SpatialFormatter.cs

```diff
@@ -44,36 +42,41 @@ public static string EncodePoint(double longitude, double latitude)
         // below to avoid extraneous allocations for adapters and to consolidate them to a single
         // source file for easier maintenance.
 
-#if EXPERIMENTAL_SPATIAL
         /// <summary>
         /// Encodes a polygon for use in OData filters.
         /// </summary>
-        /// <param name="line">The <see cref="GeoLine"/> to encode.</param>
+        /// <param name="line">The <see cref="GeoLineString"/> to encode.</param>
         /// <returns>The OData filter-encoded POLYGON string.</returns>
         /// <exception cref="ArgumentException">The <paramref name="line"/> has fewer than 4 points, or the first and last points do not match.</exception>
-        /// <exception cref="ArgumentNullException"><paramref name="line"/> or <see cref="GeoLine.Positions"/> is null.</exception>
-        public static string EncodePolygon(GeoLine line)
+        /// <exception cref="ArgumentNullException"><paramref name="line"/> or <see cref="GeoLineString.Coordinates"/> is null.</exception>
+        public static string EncodePolygon(GeoLineString line)
         {
             Argument.AssertNotNull(line, nameof(line));
-            Argument.AssertNotNull(line.Positions, $"{nameof(line)}.{nameof(line.Positions)}");
```

Comment:
So `Coordinates` will never be null? If so, great.

Reply:
`Coordinates` is an enumerable of `GeoJon` entities which are structs.